kubetest/process/process.go

```diff
@@ -170,9 +170,16 @@ func (c *Control) FinishRunning(cmd *exec.Cmd) error {
 		select {
 		case <-sigChannel:
 			log.Printf("Killing %v(%v) after receiving signal", stepName, -cmd.Process.Pid)
-			if err := syscall.Kill(-cmd.Process.Pid, syscall.SIGKILL); err != nil {
+
+			pgid, err := syscall.Getpgid(cmd.Process.Pid)
+			if err != nil {
+				pgid = cmd.Process.Pid
+			}
+
+			if err := syscall.Kill(-pgid, syscall.SIGKILL); err != nil {
```

Comment:
Do we still want `-pgid` or should we instead just do `pgid = -cmd.Process.Pid`?

Reply:
We want to target the group id and not the particular process, and researching looks like this is the right thing to do.